sklearn/model_selection/tests/test_split.py

```diff
@@ -760,14 +756,10 @@ def test_predefinedsplit_with_kfold_split():
         kf_train.append(train_ind)
         kf_test.append(test_ind)
         folds[test_ind] = i
-    ps_train = []
-    ps_test = []
     ps = PredefinedSplit(folds)
     # n_splits is simply the no of unique folds
     assert_equal(len(np.unique(folds)), ps.get_n_splits())
-    for train_ind, test_ind in ps.split():
-        ps_train.append(train_ind)
-        ps_test.append(test_ind)
+    ps_train, ps_test = zip(*ps.split())
```

Comment:
Maybe keep the original version here?
It's easier to understand and more consistent with the definition of kf_train/kf_test (and I think we'll get two tuples instead two lists here?)